pbs_exporter.sh

```diff
@@ -14,6 +14,9 @@ CURL=$(command -v curl)
 GZIP=$(command -v gzip)
 JQ=$(command -v jq)
 
+current_timestamp=$(date +%s)
+twenty_four_hours_ago=$((current_timestamp - 86400))
```

Comment:
@ebuzzz this can be simplified to something like:
`twenty_four_hours_ago=$(date +%s --date "24 hours ago")`